// promise
// const getData = new Promise((resolve, reject) => {
//   const num = Math.random() * 10;
//   console.log("generated num:", num);
//   if (num > 5) {
//     resolve({ num: num });
//   } else {
//     reject({ err: "DATA is not available" });
//   }
// });
// getData.then((data) => console.log(data)).catch((error) => console.log(error));

//promise All
